src/cbmc_starter_kit/template-for-repository/proofs/lib/summarize.py

```diff
@@ -116,4 +147,19 @@ def print_proof_results(out_file):
 
 if __name__ == '__main__':
     args = get_args()
-    print_proof_results(args.run_file)
+    with_aws = args.cloudfront_domain and args.s3_uri
+    without_aws = not args.cloudfront_domain and not args.s3_uri
+    if not with_aws and not without_aws:
+        raise argparse.ArgumentTypeError(
+            "The CloudFront domain and the S3 URI should either be specified "
+            "together or they should not be specified altogether.")
+    if with_aws:
```

Comment:
```
if without_aws:
  print_proof_results(args.run_file)
  return 0
```
then dedent the rest

Reply:
Must `s/return 0/exit(0)` because this is not inside a function. But, yes, I will use this.